src/normalize-keyframes.js

```diff
@@ -243,6 +245,10 @@
           } else if (memberValue != 'replace') {
             throw new TypeError('Invalid composite mode ' + memberValue + '.');
           }
+        } else if (member == 'easing') {
+          console.log(memberValue);
+          debugger;
+          memberValue = shared.toTimingFunction(memberValue);
```

Comment:
Good call, we should avoid making the same mistake twice. I now put it in a hidden (if Symbols are available) member.

I moved it to here because keyframe-interpolations.js doesn't read easing on the last element and normalisation is the appropriate place for validation to occur.